Documentation/Introduction/Index.rst

```diff
@@ -111,25 +119,29 @@ Each table is further defined by an array which configures how the
 system handles the table, both for the display and the processing in the
 backend. The various parts on this second level are called "sections".
 
-The general structure (looking at a single table) is as follows::
-
-   $GLOBALS['TCA']['tx_examples_haiku'] = [
-       'ctrl' => [
-           ....
-       ],
-       'interface' => [
-           ....
-       ],
-       'columns' => [
-           ....
-       ],
-       'types' => [
-           ....
-       ],
-       'palettes' => [
-           ....
-       ],
-   ];
+The general structure (looking at a single table) is as follows:
+
+
+..  code-block::php
+    :caption: EXT:my_extension/Configuration/Overrides/tx_examples_haiku.php
```

Comment:
I am not sure if "Overrides" is the correct path here: In the example you override the whole TCA for tx_examples_haiku - which is not recommended in Overrides, I assume. This is more a "how is the TCA built", not "how you should override and place it", so I suggest to remove the caption altogether.

Applies also above for "Table entries (first level)"

Reply:
Mhh, the syntax would not be correct directly in the TCA folder either. What would you recommend as a caption to make this clear?

Reply:
Like this maybe?

Reply:
Does the caption "The array looks like this" really helps? I would remove the caption, we don't need it IMHO.

Reply:
Then you cannot know if the information where to put this is only missing or if this not an example you should use